community/custard/build

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh -e 
 
 make
-make DESTDIR="$1" PREFIX=/usr install
\ No newline at end of file
+
```

Comment:
Remove this new line. New line should be below `make install`.

Reply:
done